client-device-connection/pom.xml

```diff
@@ -0,0 +1,164 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (c) 2019, 2022 Contributors to the Eclipse Foundation
+
+    See the NOTICE file(s) distributed with this work for additional
+    information regarding copyright ownership.
+
+    This program and the accompanying materials are made available under the
+    terms of the Eclipse Public License 2.0 which is available at
+    http://www.eclipse.org/legal/epl-2.0
+
+    SPDX-License-Identifier: EPL-2.0
+ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.eclipse.hono</groupId>
+        <artifactId>hono-bom</artifactId>
+        <version>2.6.0-SNAPSHOT</version>
+        <relativePath>../bom</relativePath>
+    </parent>
+    <artifactId>client-device-connection</artifactId>
+
+    <name>Device Connection client</name>
+    <description>Base classes for client for accessing device connection information in a remote cache / data grid.</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.eclipse.hono</groupId>
+            <artifactId>hono-legal</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.hono</groupId>
+            <artifactId>hono-core</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.eclipse.hono</groupId>
+            <artifactId>hono-client-common</artifactId>
+            <!-- TODO: both hono-core and hono-client-common are direct dependencies, but just declaring this one is enough -->
```

Comment:
If classes from hono-core are being referenced, should a dependency be declared on it explicitly in addition to hono-client-common or is it better to just declare a dependency on hono-client-common which transiently brings in hono-core?